courses/machine_learning/deepdive/05_review/6_deploy.ipynb

```diff
@@ -26,9 +26,13 @@
    "outputs": [],
```

Comment:
<p>should remove reference to "dherin-sandbox"</p><p> </p><p>CMLE -&gt; CAIP</p><p>Cloud ML Engine -&gt; Cloud AI Platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/730/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>